demos/hazard_mapping/train.py

```diff
@@ -0,0 +1,123 @@
+import numpy as np
```

Comment:
Please add a comment with a disclaimer that this code is provided by Ann-Kathrin and we are only responsible for adding the tracking stuff